assets/scss/components/stencil/navPages/_navPages.scss

```diff
@@ -320,7 +321,7 @@
         @include breakpoint("medium") {
             border-left: 1px solid stencilColor("navPages-subMenu-separatorColor");
             display: inherit;
-            height: $navPage-subMenu-horizontal-height;
```

Comment:
Got the changes made. Need to squash my commits still. Will do so in the morning :)